core/src/components/tab-button/tab-button.ionic.scss

```diff
@@ -86,3 +90,75 @@
 :host(.tab-button-shape-rectangular) {
   @include globals.border-radius(globals.$ion-border-radius-0);
 }
+
+// Tab Button: Badge
+// --------------------------------------------------
+
+:host ::slotted(ion-badge.badge-xxsmall),
+:host ::slotted(ion-badge.badge-xsmall),
+:host ::slotted(ion-badge.badge-small),
+:host ::slotted(ion-badge.badge-medium),
+:host ::slotted(ion-badge.badge-large),
+:host ::slotted(ion-badge.badge-xlarge) {
+  min-width: ion-vars.$tab-ionic-button-badge-size;
+  height: ion-vars.$tab-ionic-button-badge-size;
+
+  font-size: ion-vars.$tab-ionic-button-badge-font-size;
+
+  line-height: ion-vars.$tab-ionic-button-badge-line-height;
+}
+
+:host ::slotted(ion-badge:empty) {
+  @include mixins.padding(0, 0);
+
+  min-width: ion-vars.$tab-ionic-button-badge-size-emtpy;
+  height: ion-vars.$tab-ionic-button-badge-size-emtpy;
+}
+
+:host ::slotted(ion-badge) {
+  @include mixins.position($start: ion-vars.$tab-ionic-button-badge-start-position);
+}
+
+:host ::slotted(ion-badge.badge-vertical-top) {
+  @include mixins.position($top: ion-vars.$tab-ionic-button-badge-top-position);
+}
+
+:host ::slotted(ion-badge.badge-vertical-top:empty) {
+  @include mixins.position($top: ion-vars.$tab-ionic-button-badge-top-position-empty);
+}
+
+:host ::slotted(ion-badge.badge-vertical-bottom) {
+  @include mixins.position($top: ion-vars.$tab-ionic-button-badge-bottom-position);
+}
+
+:host ::slotted(ion-badge.badge-vertical-bottom:empty) {
+  @include mixins.position($top: ion-vars.$tab-ionic-button-badge-bottom-position-empty);
+}
+
+:host ::slotted(ion-badge[vertical]:not(:empty)) {
+  @include mixins.padding(
+    ion-vars.$tab-ionic-button-badge-padding-top,
+    ion-vars.$tab-ionic-button-badge-padding-end,
+    ion-vars.$tab-ionic-button-badge-padding-bottom,
+    ion-vars.$tab-ionic-button-badge-padding-start
+  );
+
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+
+  min-width: ion-vars.$tab-ionic-button-badge-size;
```

Comment:
This isn't needed. I was able to remove it on my end and verified that the sizes are still correct since we are using the correct `size="xxsmall"` prop.

```suggestion
```

Reply:
Indeed! Thank you!
Done!